Jenkinsfile

```diff
@@ -184,6 +184,17 @@ pipeline {
                         , image_suffix: "base-${PLATFORM.replaceAll('/','-')}"
                       )
                       // GCR image
+                      def additional_gcr_tags = []
+                      if (env.CHANGE_BRANCH  == "main" || env.BRANCH_NAME == "main" ) {
```

Comment:
NIT: we could extract it into a field like `isMainBranch` and reuse it in the rest of the checks.